runtime-modules/council/src/lib.rs

```diff
@@ -843,6 +851,38 @@ decl_module! {
             Ok(())
         }
 
+        /// Decrease the council total budget
+        ///
+        /// # <weight>
+        ///
+        /// ## weight
+        /// `O (1)`
+        /// - db:
+        ///    - `O(1)` doesn't depend on the state or parameters
+        /// # </weight>
+        #[weight = 10_000_000] // TODO: adjust
```

Comment:
Should we add a weight function for this dispatch

Reply:
ok added that + benchmark